src/webgpu/util/conversion.ts

```diff
@@ -1004,10 +985,9 @@ export const Type = {
   vec4f: VectorType.create(4, f32Type),
   vec4h: VectorType.create(4, f16Type),
   vec4b: VectorType.create(4, boolType),
+} as const;
 
-  mat: (cols: number, rows: number, elementType: ScalarType) =>
-    MatrixType.create(cols, rows, elementType),
-
+const kMatTypes = {
```

Comment:
Is this the same as `kAllMatrices` on line 2566?

Reply:
No, `kMatTypes` is a record of types by name, `kAllMatrices` as an array. But kAllMatrices can be generated from `kMatTypes` so made that change